recipes/esxi.rb

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+# Author:: JJ Asghar <jj@chef.io>
+# Cookbook Name:: pxe_dust
+# Recipe:: esxi
+#
+# Copyright 2016 Chef Software, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'pxe_dust::server'
+
+esxi = "/tmp/#{node['pxe_dust']['esxi_iso']}"
+
+directory '/mnt/loop' do
```

Comment:
Maybe make this `/mnt/esxi-iso` or something more specific to the purpose so you don't worry about this conflicting with other mounts or user-initiated work?